crates/net/discv5/src/metrics.rs

```diff
@@ -0,0 +1,117 @@
+//! Tracks peer discovery for [`Discv5`](crate::Discv5).
+use metrics::{Counter, Gauge};
+use reth_metrics::Metrics;
+
+use crate::config::{ETH, ETH2, OPSTACK};
+
+/// Information tracked by [`Discv5`](crate::Discv5).
+#[derive(Debug, Default, Clone)]
+pub struct Metrics {
```

Comment:
should rename to `Discv5Metrics` or something else, the `Metrics` name is overloaded